package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "fh-mbaas-api",
-  "version": "5.13.9",
+  "version": "5.13.11",
```

Comment:
@pb82 what's the reason for jumping from .9 -> 11 ? 

Also, why was there no tag created as part of the release/push to npmjs ?